recipe/meta.yaml

```
@@ -31,7 +28,7 @@ requirements:
     - m2-patch            # [win]
   host:
     - python
-    - cython >=0.29.22
+    - cython >=3.0.7
```

```suggestion
    - cython >=3.0.10
```
https://github.com/pymssql/pymssql/blob/v2.3.2/pyproject.toml#L6

Unless I'm looking at something different?

---

I was looking [here](https://github.com/pymssql/pymssql/blob/fd2baefa4e185f5a47e5bb99cc2fc6b2c7cb6e42/setup.cfg#L3).

This one is confusing because these two files plus the `setup.py` all say different things.

---

Indeed. Ok, feel free to leave it.

---

It's been a little while since I looked into it, but I believe PEP says the `pyproject.toml` will take precedence over all other `setup.py/.cfg`. I think the more recently-updated `pyproject.toml` has the pinnings we should follow.

That being said, it's kinda hard to guess upstream's intentions in their build scripts since they define requirements in 3 cascading places such that 2 out of 3 groups are never used.